x/feemarket/spec/08_client.md

```diff
@@ -0,0 +1,78 @@
+<!--
+order: 8 -->
+
+# Client
```

Comment:
this should also be a table, check the erc20 module spec for reference

Reply:
what should be a table?

Reply:
I guess the query and transaction commands commands: https://github.com/tharsis/evmos/blob/main/x/erc20/spec/08_clients.md#queries

Reply:
okay but in all the other module specs (evm, bank, gov etc) it uses the same structure with "example" and "example output"